xcode/Subconscious/Shared/Components/Detail/MemoViewerDetailView.swift

```diff
@@ -464,20 +472,45 @@ struct MemoViewerDetailModel: ModelProtocol {
         environment: Environment,
         description: MemoViewerDetailDescription
     ) -> Update<Self> {
+        guard state.address != description.address else {
+            logger.warning("Attempted to appear with same address, doing nothing")
```

Comment:
Might be a bit noisy to log?

Reply:
`logger.log` seems more appropriate here, since this is expected behavior that is being handled gracefully.